### PR TITLE
Feature/tecg 224 edit technology

### PR DESCRIPTION
**Pull request by jfmedeiros1820**

### CODE REVIEW

File: src/main/java/com/ciandt/techgallery/servlets/CreateTech.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CreateTech extends HttpServlet {
+
+  private static final long serialVersionUID = -6323934778871180015L;
+
+  @Override
+  public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/servlets/CreateTech.java]: @jfmedeiros1820 Why do we need this servlet?

File: src/main/java/com/ciandt/techgallery/persistence/model/Technology.java [hunk 2]
```diff
@@ -333,54 +342,52 @@ public int compare(Technology counter1, Technology counter2) {
...
-        Collections.sort(techList, new Comparator<Technology>() {
-          @Override
-          public int compare(Technology counter1, Technology counter2) {
-            return counter1.getName().compareTo(counter2.getName());
-          }
-        });
-        break;
-      default:
-        break;
+    case POSITIVE_RECOMMENDATION_AMOUNT:
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/persistence/model/Technology.java]: @tsr-thulio Please show @felipeaziz  how we do that using enum abstract method

File: src/main/java/com/ciandt/techgallery/service/transformer/TechnologyTransformer.java [hunk 3]
```diff
@@ -25,6 +24,7 @@ public Technology transformFrom(TechnologyTO baseObject) {
     product.setEndorsersCounter(baseObject.getEndorsersCounter());
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/service/transformer/TechnologyTransformer.java]: @tsr-thulio Seems this method has the same content as the block that is on line 34

File: src/main/webapp/js/controllers/techDetailsController.js [hunk 4]
```diff
@@ -569,5 +570,17 @@ angular.module('techGallery').controller(
       });
     }
     
+    $scope.editTechnology = function(){
+    	window.location = $scope.redirectUrl($scope.id, '/createTech');
+    }
+    
+    $scope.redirectUrl = function(techId, servlet) {
+        var protocol = location.protocol + '//';
```
Comment (brunocarneiro) [inline src/main/webapp/js/controllers/techDetailsController.js]: @tsr-thulio this is the same method as techListController.js:52
What do you think about refatoring it to an util file?

File: src/main/java/com/ciandt/techgallery/persistence/model/Technology.java (hunk 2 above)
Comment (felipeaziz) [inline src/main/java/com/ciandt/techgallery/persistence/model/Technology.java]: Improvement done.

**Pull request merged**